sync/doc/changelog.rst

```diff
@@ -1,3 +1,7 @@
+`2.1.1`
+-------
+- **Fix:** resolving name conflicts with the demo project
```

Comment:
add an empty line before the description of the change according to the formatting rules